<ui version="4.0">
 <author/>
 <comment/>
 <exportmacro/>
 <class>ChatWindowForAdmin</class>
 <widget class="QWidget" name="ChatWindowForAdmin">
 <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Form</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="messageTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="connectPushButton">
          <property name="text">
           <string>Invite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="sendPushButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <resources/>
  <connections/>
 </ui>
